modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -44,6 +48,36 @@
 }
 
 // Sidebar
+
+.has-control-menu .sidebar {
+	height: calc(100% - 56px);
+	top: 56px;
+}
+
+.sidebar {
+	height: 100%;
+	position: fixed;
+	right: -320px;
```

Comment:
doing that we will have to hardcode it in .sidebar-open .sidebar anyway:
```
.sidebar-open .sidebar {
   left: calc(100% - 320px)
}
```

I tried to avoid by doing:
```
.sidebar {
   left: 100%;
}

.sidebar-open .sidebar {
   left: initial;
   right: 0;
```

But then animation don't happen.

Don't know if I'm missing something, in the meantime I'm creating a variable for the width an using it 🤷

Reply:
Again, I just read just read #160 (comment)